system_tests/monitoring.py

```diff
@@ -176,6 +178,48 @@ def test_create_and_delete_metric_descriptor(self):
         retry_500(descriptor.create)()
         retry_404_500(descriptor.delete)()
 
+    def test_write_point(self):
+        METRIC_TYPE = ('custom.googleapis.com/tmp/system_test_example' +
+                       unique_resource_id())
+        METRIC_KIND = monitoring.MetricKind.GAUGE
+        VALUE_TYPE = monitoring.ValueType.DOUBLE
+        DESCRIPTION = 'System test example -- DELETE ME!'
+        VALUE = 3.14
+
+        client = monitoring.Client()
+        descriptor = client.metric_descriptor(
+            METRIC_TYPE,
+            metric_kind=METRIC_KIND,
+            value_type=VALUE_TYPE,
+            description=DESCRIPTION,
+        )
+
+        descriptor.create()
+        retry_404(descriptor._fetch)(client, METRIC_TYPE)
+
+        metric = client.metric(METRIC_TYPE, {})
+        resource = client.resource('global', {})
+
+        client.write_point(metric, resource, VALUE)
+
+        def _query_timeseries_with_retries():
+            def _has_timeseries(result):
+                return len(list(result)) > 0
+            retry_result = RetryResult(_has_timeseries, max_tries=7)(
+                client.query)
+            return RetryErrors(BadRequest)(retry_result)
+
+        query = _query_timeseries_with_retries()(METRIC_TYPE, minutes=5)
+        timeseries_list = list(query)
+        self.assertEqual(len(timeseries_list), 1)
+        timeseries = timeseries_list[0]
+        self.assertEqual(timeseries.metric, metric)
+        # project_id label only exists on output.
+        del timeseries.resource.labels['project_id']
+        self.assertEqual(timeseries.resource, resource)
+
+        descriptor.delete()
```

Comment:
not sure where a lot of my review comments went, but I took away the retry here, but the above test does a retry on descriptor.delete() so I thought I should copy that, but not sure.